dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -157,6 +164,99 @@ public List<Experiment> list(ExperimentFilter filter, User user) throws DotDataE
         return factory.list(filter);
     }
 
+    @Override
+    public Experiment start(String experimentId, User user)
+            throws DotDataException, DotSecurityException {
+        DotPreconditions.isTrue(hasValidLicense(), InvalidLicenseException.class,
+                invalidLicenseMessageSupplier);
+        DotPreconditions.checkArgument(UtilMethods.isSet(experimentId), "experiment Id must be provided.");
+
+        final Optional<Experiment> persistedExperimentOpt =  find(experimentId, user);
+
+        DotPreconditions.isTrue(persistedExperimentOpt.isPresent(),()-> "Experiment with provided id not found",
+                DoesNotExistException.class);
+
+        final Experiment experimentFromFactory = persistedExperimentOpt.get();
+        validatePermissions(user, experimentFromFactory,
+                "You don't have permission to archive the Experiment. "
+                        + "Experiment Id: " + persistedExperimentOpt.get().id());
+
+        DotPreconditions.isTrue(experimentFromFactory.status()!=Status.RUNNING ||
+                        experimentFromFactory.status() == Status.SCHEDULED,()-> "Cannot start an already started Experiment.",
+                DotStateException.class);
+
+        DotPreconditions.isTrue(experimentFromFactory.status()== DRAFT
+                ,()-> "Only DRAFT experiments can be started",
+                DotStateException.class);
+
+        DotPreconditions.checkState(hasAtLeastOneVariant(experimentFromFactory), "The Experiment needs at "
+                + "least one Page Variant in order to be started.");
+
+        DotPreconditions.checkState(experimentFromFactory.goals().isPresent(), "The Experiment needs to "
+                + "have the Goal set.");
+
+        final Experiment experimentToStart;
+
+        if(experimentFromFactory.scheduling().isEmpty()) {
+            final Scheduling scheduling = startNowScheduling(experimentFromFactory);
+            experimentToStart = experimentFromFactory.withScheduling(scheduling);
+        } else {
+            Scheduling scheduling = validateScheduling(experimentFromFactory.scheduling().get());
+            experimentToStart = experimentFromFactory.withScheduling(scheduling);
```

Comment:
do we need to do this? I mean in this case the experiment already has the scheduling set, right?